flutter/alds/lib/storage.dart

```diff
@@ -128,85 +103,62 @@ Future<String?> readLocationData() async {
   return await appbox.get('locdata');
 }
 
+Future<void> addNewLocation(String locationName, double latitude, double longitude) async {
+  var appbox = Hive.box('appData');
+  String? existingData = await readLocationData();
+  List<dynamic> locations = [];
+  
+  if (existingData != null) {
+    locations = json.decode(existingData);
+  }
+
+  Map<String, dynamic> newLocation = {
+    "location": locationName,
+    "position": {
+      "latitude": latitude,
+      "longitude": longitude,
+    },
+    "bluetooth": {},
+  };
+
+  locations.add(newLocation);
+  await saveLocatorData(json.encode(locations));
+}
 
 Future<void> removeLocation(SavedLocation location) async {
   var appbox = Hive.box('appData');
-  // WE ASSUME THAT LOC NAME IS UNIQUE
   String? existingData = await readLocationData();
   if (existingData != null) {
     List<dynamic> locations = json.decode(existingData);
-    // util.log("BEFORE REMOVE CALLED: $locations");
```

Comment:
You can remove line 138 and 140, though. They are just commented-out log statements.